### ISSUE TITLE
prepare R code to create PLOTS from HISTORY, including fix of numbering problem 

### ISSUE THREAD

**Issue by teixeirak** [forc-db/ForC#106]
@ValentineHerr, you're already working on this...

- rewrite [this script](https://github.com/forc-db/ForC/blob/master/scripts/Database_manipulation/Generate%20PLOTS%20from%20HISTORY/ForC_plots_from_history.m) in R to facilitate updates.

**Comment by ValentineHerr** [comment]
@teixeirak,
I see 6 duplicated lines in PLOTS (see lines 2725 to 2736).  I am assuming it is a mistake but I want to double check that I am not missing anything.

**Comment by teixeirak** [comment]
Yes, that was an error. Now fixed. This is why we're writing R code to generate PLOTS!

**Comment by ValentineHerr** [comment]
I realized that some column names in the output of the R script that converts HISTORY to PLOTS are not good. I need to deal with this so I am reopening this issue.

**Comment by ValentineHerr** [comment]
@teixeirak ,
DUKE FOREST LOBLOLLY PINE_MANAGED LOBLOLLY PINE PLANTATION
have 2 records for establishment of oldest trees in HISTORY. Is it a mistake? The code to create PLOTS from HISTORY does not handle that.

**Comment by teixeirak** [comment]
Yes, this is a mistake. Are you able to tell which can be deleted, or does it need my review?

**Comment by ValentineHerr** [comment]
@teixeirak , 
investigating the above, I found that some sites in HISTORY do not exist in MEASUREMENTS of SITES, and vice-versa. (there are also sites that are missing in SITES, compared to MEASUREMENTS).
I should probably investigate this further.... it could be that people did not enter data in all tables they should have to.... ?
